games-arcade/opentyrian/patches/opentyrian-2.1.20130907.patchset

```diff
@@ -41,5 +41,34 @@ index 3cffa49..d0f85a4 100644
  
  	printf("\nWelcome to... >> %s %s <<\n\n", opentyrian_str, opentyrian_version);
 -- 
-2.2.2
+2.21.0
+
+
+From ca8f89c7905932a43d3f2e34f8a6381236ca3e88 Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Wed, 24 Jul 2019 13:50:38 +0200
+Subject: Don't turn warnings into errors, remove -Werror
+
+
+diff --git a/Makefile b/Makefile
+index 5d1a683..8cfdeaf 100644
+--- a/Makefile
++++ b/Makefile
+@@ -1,4 +1,4 @@
+-# BUILD SETTINGS ###########################################
++ # BUILD SETTINGS ###########################################
+ 
+ ifneq ($(filter Msys Cygwin, $(shell uname -o)), )
+     PLATFORM := WIN32
+@@ -22,7 +22,7 @@ OBJS := $(SRCS:src/%.c=obj/%.o)
+ # FLAGS ####################################################
+ 
+ ifneq ($(MAKECMDGOALS), release)
+-    EXTRA_CFLAGS += -g3 -O0 -Werror
++    EXTRA_CFLAGS += -g3 -O0
```

Comment:
I think you fixed the wrong thing. from what I can see here, setting `MAKECMDGOALS` to `release` (exporting that in the recipe should do) will disable `-Werror`.